src/libs/UnreadIndicatorUpdater/index.ts

```diff
@@ -83,7 +87,7 @@ function getUnreadReportsForUnreadIndicator(reports: OnyxCollection<Report>, cur
 
 const memoizedGetUnreadReportsForUnreadIndicator = memoize(getUnreadReportsForUnreadIndicator, {maxArgs: 3});
 
-const triggerUnreadUpdate = debounce(() => {
+triggerUnreadUpdate = debounce(() => {
```

Comment:
NAB: I know you didn't add this, but it would be really nice to have a code comment which explains _why_ debouncing this is necessary. If you happen to know why, could you please add a comment?